src/LaboratoryResults-3-0/Test/_components/assert.organization.address.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<nts:component xmlns="http://hl7.org/fhir" xmlns:nts="http://nictiz.nl/xsl/testscript">
+    <nts:parameter name="warningOnly" value="false"/>
+    <action>
+        <assert>
+            <description
+                value="Confirm that Bundle contains the right address (street) for the {$resource}"/>
+            <direction value="{$direction}"/>
+            <expression value="Bundle.entry.resource.as({$resource}).address.line.extension.where(url='{$urlstreet}' and value='{$street}').exists()"/>
+            <warningOnly value="{$warningOnly}"/>
+        </assert>
+    </action>
+    
+    <action>
+        <assert>
+            <description
+                value="Confirm that Bundle contains the right address (housenumber) for the {$resource}"/>
+            <direction value="{$direction}"/>
+            <expression value="Bundle.entry.resource.as({$resource}).address.line.extension.where(url='{$urlnumber}' and value ='{$number}').exists()"/>
+            <warningOnly value="{$warningOnly}"/>
+        </assert>
+    </action>
+    
+    <action>
+        <assert>
+            <description
+                value="Confirm that Bundle contains the right address (city and postalcode) for the {$resource}"/>
+            <direction value="{$direction}"/>
+            <expression value="Bundle.entry.resource.as({$resource}).address.where(city='{$city}' and postalCode='{$postalcode}').exists()"/>
+            <warningOnly value="{$warningOnly}"/>
+        </assert>
+    </action>
+    
+    <action>
+        <assert>
+            <description
+                value="Confirm that Bundle contains the right address (type) for the {$resource}"/>
+            <direction value="{$direction}"/>
+            <expression value="Bundle.entry.resource.as({$resource}).address.extension.where(url='{$urltype}').value.coding.where(system='{$system}' and code='{$code}' and display='{$display}').exists()"/>
```

Comment:
Since this component has quite some asserts, I'd make the parameters `system`, `code` and `display` more specific. Also consider if it's really desirable to explicitly check the `display` (see similar comments elsewhere).

Reply:
@LuudSlagter do you mean using more specific names for the parameters? Such as typesystem, typecode?

Reply:
Yes!